src/main/java/org/kohsuke/github/GHRelease.java

```diff
@@ -260,7 +260,6 @@ public GHAsset uploadAsset(String filename, InputStream stream, String contentTy
      *             existing logic in place for backwards compatibility.
      */
     @Deprecated
-    @Preview
     public List<GHAsset> assets() {
```

Comment:
I was unable to find anything in the Github API documentation that confirms this is a preview API.
https://developer.github.com/v3/repos/releases/#list-release-assets